canonical/ubuntu.com
# /pricing/infra => /pricing/pro

## Done

- moved /pricing/infra to /pricing/pro
- added a redirect for the above
- updated the copy on /pricing/pro according to the copy doc: https://docs.google.com/document/d/1Ynk0iLa9hjrdich6ATTsybtPbDwW_k4vRI0nk_HNhcI/edit#heading=h.l83dv5879ajz

## QA

- Visit https://ubuntu-com-12083.demos.haus/pricing/infra
- See that you are redirected to https://ubuntu-com-12083.demos.haus/pricing/pro
- See that the page matches the copy doc

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6016
